docs/source/troubleshooting.rst

```diff
@@ -226,9 +226,32 @@ On Ubuntu / Debian:
 .. code-block:: bash
 
     /usr/lib/python2.7/dist-packages/st2common/bin/st2-self-check
-    
+
 On RedHat / Fedora:
 
 .. code-block:: bash
 
     /usr/lib/python2.7/site-packages/st2common/bin/st2-self-check
+
+
+Support
+-------
+
+Support is available to you via `email <support@stackstorm.com/>`_ or via
+`slack-community <https://stackstorm.typeform.com/to/K76GRP)/>`_.
+
+Before asking for help, please have the following list ready.
+
+1. st2 version. (``st2 --version``)
+2. st2client version. (``pip show st2client``)
+3. OS version. (Cent OS 7. ``uname -a``)
+4. Python version. (``python --version``)
+5. A brief description of the problem.
+6. Any logs you have collected. (Please use `Pastebin <http://pastebin.com/>`_ or `gists <https://gists.github.com/>`_.)
+
```

Comment:
Any relevant logs (please use .... ) Or submit the full debug information (link to prev section).